x/stake/types/cross_stake.go

```diff
@@ -83,30 +68,17 @@ type CrossStakeRedelegateSynPackage struct {
 	Amount  *big.Int
 }
 
-type CrossStakeRedelegateAckPackage struct {
-	EventType CrossStakeEventType
-	DelAddr   sdk.SmartChainAddress
-	ValSrc    sdk.ValAddress
-	ValDst    sdk.ValAddress
-	Amount    *big.Int
-	ErrorCode uint8
-}
-
-func NewCrossStakeRedelegationAckPackage(synPack *CrossStakeRedelegateSynPackage, eventType CrossStakeEventType, errCode uint8) *CrossStakeRedelegateAckPackage {
-	return &CrossStakeRedelegateAckPackage{eventType, synPack.DelAddr, synPack.ValSrc, synPack.ValDst, bsc.ConvertBCAmountToBSCAmount(synPack.Amount.Int64()), errCode}
-}
-
 type CrossStakeDistributeRewardSynPackage struct {
 	EventType CrossStakeEventType
-	Amount    *big.Int
 	Recipient sdk.SmartChainAddress
+	Amount    *big.Int
 }
 
 type CrossStakeDistributeUndelegatedSynPackage struct {
```

Comment:
For the `CrossStakeRefundPackage`, let us put `amount` behind `Recipient` as well

Reply:
Done